### ISSUE TITLE
DeprecationWarning during pytests

### ISSUE THREAD

**Issue by DuckNrOne** [tBuLi/symfit#260]
While executing the first converted pytest (test_argument.py), I got a warning, that some collections and syntax of symfit won't be supported in the future. I just wanted to point this out to you :)

If you want to look into the code or the log-file, I'll link the [pytest](https://github.com/DuckNrOne/symfit/blob/ConvertToPyTest/tests/test_argument.py) and the [log-file](https://github.com/DuckNrOne/symfit/blob/ConvertToPyTest/tests/test_argument.log).

**Comment by tBuLi** [comment]
Thanks for reporting back :).

If you want, you could do the following: make an assert that the current syntax raises a deprecation warning, since they are raised for a reason, and then do the rest of the test properly with new syntax so that it still tests for the same thing.

But perhaps it is best to first complete the translation, and to then make improvements, and just start an issue for every test you think is faulty so we don't forget about it.

**Comment by DuckNrOne** [comment]
You're right. I'll concentrate on the translation and make a pull request after it.
